patches/chrome-browser-resources-settings-settings.js.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/settings/settings.js b/chrome/browser/resources/settings/settings.js
+index bf8087a0229070d042576c4d0081b11f4097b11b..1a667ba004435c4e4318207edb02d8e40d84fae1 100644
+--- a/chrome/browser/resources/settings/settings.js
++++ b/chrome/browser/resources/settings/settings.js
+@@ -3,7 +3,7 @@
+ // found in the LICENSE file.
+ 
+ import './settings_ui/settings_ui.js';
+-
++import './brave_page_visibility.js';
```

Comment:
This will move to a single `brave_settings_overrides.js` when I get to that, next...